### ISSUE TITLE
Segmentation in the window close

### ISSUE THREAD

**Issue by kjdev** [geommer/yabar#77]
Environment: Fedora release 23 (Twenty Three)

---

Segmentation occurs when you close the window.

```
Program received signal SIGSEGV, Segmentation fault.
ya_handle_prop_notify (ep=ep@entry=0x646d20) at src/ya_exec.c:503
503                     ya_exec_intern_ewmh_blk(ewblk->blk);
```

**Comment by geommer** [comment]
Hi @kjdev, `YABAR_TITLE` internal block is currently buggy and my segfault on certain cases. Please use [xtitle](https://github.com/baskerville/xtitle) as an external block until this bug get fixed.

**Comment by kjdev** [comment]
segfault no longer occurs when you change to `xtitle`.

**Comment by geommer** [comment]
Yeah, I also just merged a fix #69 contributed by @abarbu that may fix this bug. You may try to rebuild yabar after pulling these latest commits and test whether `YABAR_TITLE` works well again under all conditions.

**Comment by kjdev** [comment]
emacs, firefox, .. of the title will be displayed along with the focus.
xterm, mlterm of the title does not appear when you focus.

```
% xterm -T title
```

```
ya_title: {
  exec: "YABAR_TITLE";
```

**Comment by kjdev** [comment]
xterm of the title will not be displayed.

```
% git log -1
commit 82e6a2789eecf8a69191510548bc8069727ebdfc
Author: geommer <geommer@gmail.com>
Date:   Tue Jun 14 03:12:08 2016 +0200

    fast improvement for fix #69
% ./yabar -c examples/internal1.conf
```

**Comment by Lesik** [comment]
I think this instability of `YABAR_TITLE` should be explicitly pointed out in the README. New users might think the preferred way of getting the window title to be `YABAR_TITLE` (since it's an internal block instead of an external command, makes sense one would prefer this one) and be disappointed by the resulting segmentation faults.

Since there was no new yabar release since @geommer's/@abarbu's fix, distributions that package yabar still distribute the old, unfixed version (as distributions don't build from master branch).
